.github/workflows/tflint.yml

```diff
@@ -0,0 +1,17 @@
+name: run-tflint
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  run-tflint:
+    name: Run tflint
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - run: brew install tflint
```

Comment:
Does this actually work? (rhetorical question)

Reply:
more specifically does the github actions vm this runs on actually have brew installed?

Reply:
oh it, actually does haha. nevermind